src/Analysis/GWASV2/Steps/SelectOutcome/SelectOutcome.jsx

```diff
@@ -39,12 +39,13 @@ const SelectOutcome = ({
       return (
         <div className='select-outcome-container'>
           <CustomDichotomousCovariates
+            dispatch={dispatch}
             studyPopulationCohort={studyPopulationCohort}
             outcome={outcome}
             handleClose={() => {
               setSelectionMode('');
             }}
-            dispatch={(chosenOutcome) => {
+            handleSelect={(chosenOutcome) => {
```

Comment:
Should the error message for Selecting Covariates be identical to the error message for Selecting an Outcome? We might want them to be different so it is more clear to the user what they've done to cause an error

Idea: the error message information could be passed as a prop here instead of it being hardcoded in CustomDichotomousCovariates. Error messages could all live in a single file that could be imported i.e. ErrorMessages.js

Reply:
> If the user creates an error in SelectOutcome and then also in Select Covariates, then the user fixes the error in Select Covariates - i think the error for the SelectOutcome would be dismissed when it shouldn't be

Good point. I'll make the messages unique.

Reply:
As discussed, this will take some more work, given the components are reused in different steps, while they're not aware of which step they are in. I'll create a separate ticket for refining/improving the messages.

Reply:
https://ctds-planx.atlassian.net/browse/VADC-413